Tests/SWBTaskExecutionTests/SwiftHeaderToolTaskActionTests.swift

```diff
@@ -0,0 +1,135 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift open source project
+//
+// Copyright (c) 2025 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Testing
+import SWBUtil
+import SWBTestSupport
+import SWBTaskExecution
+
+struct SwiftHeaderToolTaskActionTests: CoreBasedTests {
+
+    /// Utility function to create and run a `SwiftHeaderToolTaskAction` and call a block to check the results.
+    private func testSwiftHeaderToolTaskAction(archs: [String], validate: (ByteString) -> Void) async throws {
+        guard !archs.isEmpty else {
+            Issue.record("Must pass at least one architecture.")
+            return
+        }
+
+        let fs = PseudoFS()
+
+        // Build the command line and write the input files.
+        try fs.createDirectory(Path("/Volumes/Inputs"), recursive: true)
```

Comment:
This crashes on Windows: SWBUtil/FSProxy.swift:1007: Precondition failed: input path '\Volumes' must be absolute

For any tests constructing absolute paths, we should generally be using a pattern like this, to account for the differences in path structures across platforms:


```suggestion
        try fs.createDirectory(.root.join("Volumes/Inputs"), recursive: true)
```

On Unix that'll be the same as /Volumes/Inputs but on Windows it'll prepend the current drive so you'll get C:\Volumes\Inputs which is a correctly formed absolute path.

Alternatively, you can use withTemporaryDirectory and just make all your paths relative to the path it gives you in the closure.